l10n_it_vat_registries_rc/report/report_registro_iva.xml

```diff
@@ -26,14 +36,13 @@
             <t t-set="original_supplier" t-value="original_invoice.partner_id" />
             <t
                     t-set="reference"
-                    t-value="original_invoice.ref or original_invoice.name or ''"
+                    t-value="(original_invoice.ref or original_invoice.name or '') + ' - ' + (move.rc_purchase_invoice_id.ref or move.rc_purchase_invoice_id.name or '')"
```

Comment:
Se non ci sono né `.ref` né `.name`, viene aggiunto solo `" - "`: è possibile non aggiungere nulla in questo caso?

Reply:
Non mi risultano casi dove `name` di `account.move` possa essere vuoto, quindi potremmo non preoccuparcene

Reply:
> Non mi risultano casi dove `name` di `account.move` possa essere vuoto, quindi potremmo non preoccuparcene

Il campo `account.move.name` non è `required`:
https://github.com/odoo/odoo/blob/5d4a24afa983a82357d76b5d6a09750b26ce3de7/addons/account/models/account_move.py#L86-L92
quindi può essere vuoto.
Può succedere ad esempio quando l'utente svuota il campo manualmente, come viene suggerito da Odoo nei casi in cui la sequenza non è valida:
https://github.com/odoo/odoo/blob/5d4a24afa983a82357d76b5d6a09750b26ce3de7/addons/account/models/sequence_mixin.py#L110

Reply:
Intendo che in questo caso sto stampando un'autofattura e recuperando i dati della fattura originale. Se esiste l'autofattura vuol dire che la fattura originale è validata e una fattura validata non mi risulta possa non avere un numero